README.md

```diff
@@ -188,6 +190,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Qt 4 ile C++ GUI Programlama](https://ufukuzun.wordpress.com/yayinlarim/qt-4-ile-c-gui-programlama/) - [Ufuk Uzun](https://ufukuzun.wordpress.com/)
 - [C++ - learnXinYminutes](https://learnxinyminutes.com/docs/tr-tr/c++-tr/)
 - [C++ En İyi Uygulamalar](https://p1v0t.gitbooks.io/cppbestpractices/content/)
+- [Türkçe C++ Kaynakları](https://github.com/vimevim/turkce-cpp-kaynaklari)
```

Comment:
ve bu zaten #388 ile eklenmedi mi? Neden burada tekrar goruyoruz? `master`'in son halini aldiginizdan emin olur musunuz?